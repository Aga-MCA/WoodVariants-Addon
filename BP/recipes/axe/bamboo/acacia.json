{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "bamboo_acacia_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:acacia_stick"
      },
      "P": {
        "item": "aga_wood_variants:bamboo_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:bamboo_acacia_axe"
    }
  }
}