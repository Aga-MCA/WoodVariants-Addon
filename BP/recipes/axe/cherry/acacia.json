{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "cherry_acacia_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:acacia_stick"
      },
      "P": {
        "item": "aga_wood_variants:cherry_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:cherry_acacia_axe"
    }
  }
}