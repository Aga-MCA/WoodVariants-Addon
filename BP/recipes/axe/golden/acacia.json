{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_acacia_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:acacia_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_acacia_axe"
    }
  }
}